docs/documentation/head/connect.md

```diff
@@ -496,6 +496,10 @@ Connection conn = DriverManager.getConnection(url);
 	In default mode (disabled) hosts are connected in the given order. 
 	If enabled hosts are chosen randomly from the set of suitable candidates.
 
+* **localSocketAddress** = String
+
+	Hostname or IP address given to explicitly configure the interface that the driver will bind the client side of the TCP/IP connection to when connecting.
```

Comment:
Could you please add something to the use cases? For instance: "If you have multiple network cards and or IP addresses that can route to the same database host, then you might want to specify which IP to use for the source IP".

Reply:
Well, just for the records – our use case is that our database server system has firewall rules and not all addresses of our client system are enabled. If the system chooses (randomly/hidden methodology) an address, there is a chance that the client cannot connect.

Reply:
It is about the own address, not the remote address. My colleagues from the NOC department assign multiple addresses to each system. One is a system specific address, another is a service specific address. If – for example due to a hardware failure – a service needs to move to a different machine, the service address is moved along with the service itself. Thus the firewalls of the systems to which the moved service connects do not need to be reconfigured.

Reply:
I think adding "this can be used to explicitly choose which interface the client will connect to" is useful. However I now know how this works so it seems rather obvious :)

I'd also like to see much more documentation in the actual usage of the code in PGStream. 

Thanks!